automate.py

```diff
@@ -0,0 +1,113 @@
+
```

Comment:
this script needs a more descriptive name than `automate`
Perhaps `automate_widgets_screenshots.py` or just `automate_screenshots.py`

Also, I think we should move this into `util/` directory.
I'd like to move all our utility scripts there.